# For some reasons I get build error for the following pages
#
# Error: Export encountered errors on following paths:
#         /ar/cookbook/cosmos
#         /es/cookbook/cosmos
#         /ja/cookbook/cosmos
#         /ko/cookbook/cosmos
#         /vi/cookbook/cosmos
#         /zh/cookbook/cosmos
#
# While visiting locally /ar/cookbook/cosmos
# useRouter() returs route which is /en/cookbook/cosmos

diff --git a/dist/use-internals.mjs b/dist/use-internals.mjs
index 43482104268d7086e219a0704497f2b3bbe8ddb2..ce08afb3cd60c99ddd3fc7b165faf8d32ebf0033 100644
--- a/dist/use-internals.mjs
+++ b/dist/use-internals.mjs
@@ -17,7 +17,7 @@ function useInternals() {
       };
     }, [route, __nextra_internal__.refreshListeners, rerender]);
   }
-  const context = __nextra_internal__.context[route];
+  const context = __nextra_internal__.context[__nextra_internal__.route]
   if (!context) {
     throw new Error(
       `No content found for the current route. This is a Nextra bug.`
